src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -248,21 +246,11 @@ protected void onStartup() {
      * Must be called after data source initialization.
      */
     private void checkToolRequirements() {
-        if ( requiresReads() && ! hasReads() ) {
```

Comment:
where are those checks now? (ie where do we check that you reads are provided if they are required?)